cmd/zdb/zdb.c

```diff
@@ -7509,6 +7509,18 @@ mos_leak_log_spacemaps(spa_t *spa)
 		mos_obj_refd(sls->sls_sm_obj);
 }
 
+static void
+traverse_errorlog(objset_t *mos, uint64_t errlog)
+{
+	zap_cursor_t zc;
+	zap_attribute_t za;
+	for (zap_cursor_init(&zc, mos, errlog);
+	    zap_cursor_retrieve(&zc, &za) == 0; zap_cursor_advance(&zc)) {
```

Comment:
nit: the convention used elsewhere is to put `zap_cursor_advance(&zc)` on its own line.